package/samplers/synetune_sampler/syne_tune_sampler.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import Any
+from typing import Optional
+
+from optuna.distributions import BaseDistribution
+from optuna.distributions import CategoricalDistribution
+from optuna.distributions import FloatDistribution
+from optuna.distributions import IntDistribution
+from optuna.study import Study
+from optuna.trial import FrozenTrial
+from optuna.trial import TrialState
+import optunahub
+from syne_tune.backend.trial_status import Trial
+from syne_tune.config_space import choice
+from syne_tune.config_space import finrange
+from syne_tune.config_space import lograndint
+from syne_tune.config_space import loguniform
+from syne_tune.config_space import randint
+from syne_tune.config_space import uniform
+from syne_tune.optimizer.baselines import BORE
+from syne_tune.optimizer.baselines import CQR
+from syne_tune.optimizer.baselines import KDE
+from syne_tune.optimizer.baselines import RandomSearch
+from syne_tune.optimizer.baselines import REA
+from syne_tune.optimizer.scheduler import AskTellScheduler
+
+
+# Currently supported methods
+searcher_cls_dict = {
+    "random_search": RandomSearch,
+    "bore": BORE,
+    "kde": KDE,
+    "regularized_evolution": REA,
+    "cqr": CQR,
+}
+
+
+class SyneTuneSampler(optunahub.samplers.SimpleBaseSampler):
+    def __init__(
+        self,
+        search_space: dict[str, BaseDistribution],
+        mode: str = "min",
+        metric: str = "mean_loss",
+        searcher_method: str = "random_search",
+        trial_mapping: Optional[dict] = None,
+        searcher_kwargs: Optional[dict] = None,
+    ) -> None:
+        super().__init__(search_space)
```

Comment:
Can you add a doc string comment here? Would make sense to specify which search-space type and method are supported.